packages/picker/src/Picker.ts

```diff
@@ -371,6 +367,11 @@ export class PickerBase extends SizedMixin(Focusable) {
         if (changes.has('selectedItem')) {
             this._selectedItemContent = undefined;
         }
+        if (this.selects) {
+            // Always force `selects` to "single" when set.
+            // TODO: Add support functionally and visually for "multiple"
```

Comment:
For `Picker` itself we only want Single, right? It's just adding support to `PickerBase`? (it seems like it might be good to eventually break PickerBase out to a separate file and possibly rename)

Reply:
Yeah, I've been thinking about whether `Picker` is the base or is `Action Menu` the base, and when we reconcile all that in the near future with `Combobox` I'll be sure to simplify/clarify this.